worlds/yoshisisland/Rom.py

```diff
@@ -0,0 +1,1231 @@
+import hashlib
+import os
+import Utils
+from worlds.Files import APDeltaPatch
+from settings import get_settings
+from typing import TYPE_CHECKING
+
+from .Options import YoshiColors, BowserDoor, PlayerGoal, MinigameChecks
+
+if TYPE_CHECKING:
+    from . import YoshisIslandWorld
+USHASH = "cb472164c5a71ccd3739963390ec6a50"
+ROM_PLAYER_LIMIT = 65535

```

Comment:
![image](https://github.com/ArchipelagoMW/Archipelago/assets/3189725/ed9c7e63-ac1f-45ce-9af2-5bc665dd768e)
Unused?

Reply:
I think so, need it removed?